src/types/redis_hash.cc

```diff
@@ -25,11 +25,12 @@
 #include <algorithm>
 #include <cctype>
 #include <cmath>
+#include <random>
 #include <utility>
 
 #include "db_util.h"
 #include "parse_util.h"
-
+#include "time_util.h"
```

Comment:
Leave a blank line here, the same as the end of namespace and between functions.   :)